jetty-start/src/main/java/org/eclipse/jetty/start/BaseBuilder.java

```diff
@@ -91,7 +91,10 @@ else if (args.isCreateFiles())
             if (localRepoDir != null)
             {
                 // Use provided local repo directory
-                fileInitializers.add(new MavenLocalRepoFileInitializer(baseHome,localRepoDir,args.getMavenLocalRepoDir()==null));
+                fileInitializers.add(new MavenLocalRepoFileInitializer(baseHome, //
```

Comment:
empty comments?

Reply:
well I just wanted to avoid a too long line :-)